include/dlr.h

```diff
@@ -250,6 +251,46 @@ int GetDLRNumOutputs(DLRModelHandle* handle, int* num_outputs);
 DLR_DLL
 int GetDLROutputSizeDim(DLRModelHandle* handle, int index, int64_t* size,
                         int* dim);
+
+/*!
+ \brief Check if metadata file is found in the compilation artifact
+ \param handle The model handle returned from CreateDLRModel().
+ \param has_metadata The pointer to save boolean value to indicate the presence of metadata file.
+*/
+DLR_DLL int GetDLRHasMetadata(DLRModelHandle* handle, bool* has_metadata);
+
+/*!
+ \brief Gets the ouput node names of the uncompiled model from the metadata file
+  \param handle The model handle returned from CreateDLRModel().
+  \param names The pointer to save array containing output node names.
+*/
+DLR_DLL int GetDLROutputName(DLRModelHandle* handle, const int index, char** name);
+
+/*!
+ \brief Gets the ouput node index using the node name
+  \param handle The model handle returned from CreateDLRModel().
+  \param name The pointer pointing to the ouput node name.
+  \param index The pointer to save the corresponding index of the output node.
+*/
+DLR_DLL int GetDLROutputIndex(DLRModelHandle* handle, char* name, int* index);
+
+/*!
+ \brief Gets the output of the node of the given name from the model.
+ \param handle The model handle returned from CreateDLRModel().
+ \param name The name of the ouput node.
+ \param out The pointer to save the output data. This should be a pointer to an
+ array of size "size" from GetDLROutputSizeDim(). \return 0 for success, -1 for
+ error. Call DLRGetLastError() to get the error message.
+ */
+DLR_DLL int GetDLROutputByName(DLRModelHandle* handle, char* name, float* out);
+
+/*!
+ \brief Gets the ouput node dtypes from the metadata file
+ \param handle The model handle returned from CreateDLRModel().
+ \param dtypes The pointer to save array containing output node dtypes.
+*/
+DLR_DLL int GetDLROuputDataType(DLRModelHandle* handle, const int index, char** dtype);
```

Comment:
`GetDLROutputType` was added in PR176

Reply:
@apivovarov I think we should read output related data from the metadata file for tvm for the sake of consistency.